README.md

```diff
@@ -1,18 +1,18 @@
 # Teleport
 
 Teleport is an identity-aware, multi-protocol access proxy which understands
-SSH, HTTPS, Kubernetes API, MySQL and PostgreSQL wire protocols.
+SSH, HTTPS, Kubernetes API, MySQL, and PostgreSQL wire protocols.
```

Comment:
I'm not a fan of Oxford commas with 'and', might just be a personal preference thing.

```suggestion
SSH, HTTPS, Kubernetes API, MySQL and PostgreSQL wire protocols.
```

Reply:
lol, I don't really care one way or the other. I just have been used to it, no problem dropping it :P